src/relationships.test.ts

```diff
@@ -0,0 +1,53 @@
+import { getScraper } from './test-utils';
```

Comment:
Note that this uses the scrape builder from PR #59. Will need to be changed if parent PR not merged.